apps/sms/test/marionette/composer_test.js

```diff
@@ -52,51 +52,41 @@ marionette('Messages Composer', function() {
   });
 
   suite('Preserve message input while navigating', function() {
-    var composer, inbox, conversation;
+    var inboxView, conversationView;
     var message = 'test message';
 
-    function waitForInbox() {
-      client.helper.waitForElement(inbox.mmsConversation);
-    }
-
-    function createMMSConversation() {
-      inbox.navigateToComposer();
-      messagesApp.addRecipient('a@b.c');
-      messagesApp.addRecipient('s@p.c');
-      composer.messageInput.sendKeys('MMS thread.');
-      messagesApp.send();
-    }
-
     setup(function() {
-      conversation = messagesApp.Conversation;
-      composer = messagesApp.Composer;
-      inbox = messagesApp.Inbox;
-
       messagesApp.launch();
-      createMMSConversation();
-      messagesApp.performHeaderAction();
-      waitForInbox();
-      inbox.mmsConversation.tap();
 
-      composer.messageInput.tap();
-      composer.messageInput.sendKeys(message);
+      inboxView = new InboxView(client);
+
+      var newMessageView = inboxView.createNewMessage();
+      newMessageView.addNewRecipient('a@b.c');
+      newMessageView.addNewRecipient('s@p.c');
+      newMessageView.typeMessage('MMS thread.');
```

Comment:
Sorry, I feel myself blind :( I don't see that any test in "Preserve message input while navigating" suite in master creates attachment [1]?

[1] https://github.com/mozilla-b2g/gaia/blob/master/apps/sms/test/marionette/composer_test.js#L54-L101